server/portal/apps/users/views.py

```diff
@@ -130,6 +130,11 @@ def get(self, request):
         """
         data = get_allocations(request.user.username)
 
+        # This line iterates through all the projects in the active allocations and filters out the ones that are excluded in settings.
+        filtered_projects = [project for project in data["active"] if project["projectName"]not in settings.ALLOCATIONS_TO_EXCLUDE]
```

Comment:
nit: add space before `not`
```suggestion
        filtered_projects = [project for project in data["active"] if project["projectName"] not in settings.ALLOCATIONS_TO_EXCLUDE]
```